src/util.js

```diff
@@ -638,6 +638,10 @@ export default {
     return navigator.hardwareConcurrency || 1;
   },
 
+  getUseElliptic: function() {
+    return config.use_indutny_elliptic;
+  },
+
```

Comment:
I'm not sure this function is really necessary - especially once you're only checking it in one place, you can just check
```js
if (!config.use_indutny_elliptic) {
```
directly.

Then, in the test suite where you're checking this in a bunch of places, I'm not sure it's really necessary? We're never calling the test suite with `config.use_indutny_elliptic = false`, so it should always be available in the test suite.

Reply:
I have this function, so I could avoid importing config in bunch of places. I think it is valid use-case. 
Regarding tests, since we have option to disable elliptic, let's have test suite which would work with this option.

Reply:
> I have this function, so I could avoid importing config in bunch of places.

That's why I mentioned "especially once you're only checking it in one place", because then you only have to import config in one place. I don't think that importing `util` instead of importing `config` is better, when it is `config` that you really need to check.

The test suite already uses `config` in lots of places, you could search for `config` in the test directory to check.

Reply:
removed